backend.native/compiletest/src/main/kotlin/org/jetbrains/kotlin/compiletest/DistProperties.kt

```diff
@@ -0,0 +1,17 @@
+package org.jetbrains.kotlin.compiletest
+
+import java.nio.file.Path
+import java.nio.file.Paths
+
+
+object DistProperties {
+    private val isWindows: Boolean = requireProp("os.name").startsWith("Windows")
```

Comment:
Please reuse same logic as in other places: 
TargetManager.host.family == Family.WINDOWS.

Reply:
Resolved.